plugins/inputs/smart/smart.go

```diff
@@ -436,14 +446,14 @@ func parseTemperature(fields, deviceFields map[string]interface{}, str string) e
 }
 
 func init() {
-	m := Smart{}
+	m := NewSmart()
 	path, _ := exec.LookPath("smartctl")
 	if len(path) > 0 {
 		m.Path = path
 	}
 	m.Nocheck = "standby"
 
 	inputs.Add("smart", func() telegraf.Input {
-		return &m
+		return m
```

Comment:
This was probably a bug before, but we need this function to always return a new instance for it to work correctly if the plugin is specified multiple times.

Reply:
Thanks Daniel. How does it look now?